### PR TITLE
Modal: New prop for aligning modal to top

### PR DESCRIPTION
**Pull request by HalvorHaugan**
Also fixes issue where `polyfillClassName` wasn't being set correctly.

### CODE REVIEW

File: .changeset/tidy-boats-rescue.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@navikt/ds-react": minor
+"@navikt/ds-css": minor
+---
+
+Modal: New prop for aligning modal to top
```
Comment (KenAJoh) [inline .changeset/tidy-boats-rescue.md]:
```suggestion
Modal: New prop `placement` for anchoring modal to the top or center of viewport. 
```

**Review by KenAJoh — DISMISSED** [review]

File: .changeset/tidy-boats-rescue.md (hunk 1 above)
Comment (HalvorHaugan) [reply .changeset/tidy-boats-rescue.md]: When I read "top or center", to me it sounds like anchoring to the center is a new feature. (Also, does it make sense to say "anchored to center"?)
Counter-suggestion:
```suggestion
Modal: New prop `placement` for anchoring the modal to the top of the viewport.
```

Comment (HalvorHaugan) [reply .changeset/tidy-boats-rescue.md]: Or do you think we should mention `center` somehow? 🤔

Comment (KenAJoh) [reply .changeset/tidy-boats-rescue.md]: Can omit "center" 👍 Anchoring might be to technical, so "aligning" or "placing" works 2.

**Review by KenAJoh — APPROVED** [review]

**Pull request merged**